doc/tools/flake8.md

```diff
@@ -0,0 +1,16 @@
+# flake8
+
+## install flake8
+
+ Install the dependencies and the package [here](../../server/README.md#how-to-run-package-version)
+
+## how to play
+
+ Run flake8:
+```sh
+flake8
+```
+ 
+ Check the meaning of the error [here](https://www.flake8rules.com/)
+
+ And then change your files to respect PEP8
```

Comment:
Maybe it should be in the server folder ?

Reply:
no again a file that explain every linter